Gemfile

```diff
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
-ruby "2.5.3"
+ruby "2.5.1"
```

Comment:
Is there a good reason for why the `ruby "2.5.3"` version was replaced with an older `ruby "2.5.1"` version?

Reply:
No specific reason, I was having trouble installing `ruby "2.5.3"`